src/classes/BDI_DataImportService.cls

```diff
@@ -878,6 +878,18 @@ global with sharing class BDI_DataImportService {
                 return false;
             }
 
+            // If the destination field is a Boolean and the DI field intermediate requires interpretation to Boolean
+            // This allows for mapping checkboxes to a tri-state picklist - None, True, False - or a text with either 
```

Comment:
Will this allow for mapping from picklists with any values?  if so we might not want to specifify "None" here, since admins can configure whatever PL values they want.

Reply:
None just means null or no value ... it's important for the behavior but I've been struggling with how to type that to make it clear that it's not an actual picklist option with text "none" but that there IS a valid option in the picklist that is neither true or false and evaluates to null

Reply:
Maybe I should just say "null" ... 🤔

Reply:
And if the PL values aren't parse-able to a boolean value with Boolean.valueOf() it will fail

Reply:
I see!  Ok, yeah, very cool.